### ISSUE TITLE
[TF2] [Feature Request] TF2's max volume is too high for current standards. Reduce it.

### ISSUE THREAD

**Issue by Ashetf2** [ValveSoftware/Source-1-Games#4797]
![image](https://user-images.githubusercontent.com/63692647/221370196-f7832117-5480-426e-a4d7-faee898d8d22.png)

This is the wave form of a gameplay in TF2 at max volume. As you can see, during combat the wave constantly reaches the highest point (0 dB). This is not good, it is too high. Since at first launch the game is at max volume, this causes most of new players to have to reduce it, even at less than half.

![image](https://user-images.githubusercontent.com/63692647/221370426-23aff713-0e71-4f1e-bc57-0c9cb79221df.png)

In comparison, a wave form of a OW2 gameplay, a more recent game. The wave almost never reachest the highest point.

This issue should be addressed to protect users' hearing. Many players may not be aware of this and may be playing the game at a dangerous volume level, especially kids. Hearing loss is a serious problem that can't be ignored.

**Comment by kisak-valve** [comment]
Hello @Ashetf2, this is a product of an era where lower quality speakers attached to computers was the norm, and it's still typical with the built-in speakers of many laptop models. It's counterproductive to make the experience on that kind of hardware worse.

Let's use this feature request to only focus on reducing the default master volume.

**Comment by Ashetf2** [comment]
You can see this issue in action here, where a content creator just launched the game for the first time
https://youtu.be/xAO22KElYDo?t=98

**Comment by AdiRege1** [comment]
i agree this shit is fucking ass i wish there was a way to lower the audio of guns or explosions individually

**Comment by Ashetf2** [comment]
Another case has happened today. People complaining about max volume being too high

**Comment by Ashetf2** [comment]
I think that a more viable option would be to change the default volume to 0.35, and change the slider in the audio settings to have a max value of 0.35, so, in optionssubaudio.cpp this should be changed to:

`m_pSFXSlider = new CCvarSlider( this, "SFXSlider", "#GameUI_SoundEffectVolume", 0.0f, 0.35f, "volume" );`

**Comment by Ashetf2** [comment]
> I think that a more viable option would be to change the default volume to 0.35, and change the slider in the audio settings to have a max value of 0.35, so, in optionssubaudio.cpp this should be changed to:
> 
> `m_pSFXSlider = new CCvarSlider( this, "SFXSlider", "#GameUI_SoundEffectVolume", 0.0f, 0.35f, "volume" );`

I had tested this and it indeed works

https://github.com/user-attachments/assets/cdde3abb-ab5e-476f-ac68-2c11a260ad05

Besides that line, in COptionsSubAudio::OnResetData() this also needs to be added to properly reset the range to max 0.35

m_pSFXSlider->SetMinMaxValues(0.0f, 0.35f, false);
